setup.py

```diff
@@ -26,7 +26,7 @@
     'google_cloud' : ['google-auth', 'google-api-python-client'],
     'gssapi' : ['python-gssapi'],
     'azure' : ['azure', 'msrestazure'],
-    'workqueue': ['work_queue'],
+    'workqueue': ['conda_pack, stdlib_list'],
```

Comment:
this should be a python list of python strings, not a single string with a non-python list inside it.